packages/adapter-knex/lib/adapter-knex.js

```diff
@@ -102,6 +102,57 @@ class KnexAdapter extends BaseKeystoneAdapter {
     return fkResult;
   }
 
+  async _createAdjacencyTable({ tableName, relationshipFa, leftListAdapter }) {
+    // Create an adjacency table for the (many to many) relationship field adapter provided
+    const dbAdapter = this;
+    try {
+      console.log(`Dropping table ${tableName}`);
+      await dbAdapter.schema().dropTableIfExists(tableName);
+    } catch (err) {
+      console.log('Failed to drop');
+      console.log(err);
+      throw err;
+    }
+
+    // To be clear..
+    const leftPkFa = leftListAdapter.getPrimaryKeyAdapter();
```

Comment:
If we're going to be clear can we be clearer? `leftPrimaryKey(what is 'Fa')`

Reply:
The cleanliness of this code is out of scope for this PR. It's all just moving around here, and it's almost definitely going to change in future PRs.